client/components/tinymce/iframe.scss

```diff
@@ -103,11 +103,14 @@ a img {
 	}
 }
 
-.wpview-wrap iframe {
-	display: block;
+.wpview-wrap iframe:not(.wpview-type-video) {
 	width: 100%;
 }
 
+.wpview-wrap iframe {
+  display: block;
```

Comment:
Mixed whitespace (this should be tabs, not spaces).

Additionally, you may consider leveraging nesting for the pseudo-class `:not` selector ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines)), i.e.

```sass
.wpview-wrap iframe {
	display: block;

	&:not( .wpview-type-video ) {
		width: 100%;
	}
}
```

Also note spaces between parens ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#spacing)).